### ISSUE TITLE
Mulnbapi issue

### ISSUE THREAD

**Issue by openmul** [openmul/openmul#11]
Reported by Антон Ефлюков

During use, the module "mulnbapi" error occurs "AttributeError: 'SwigPyObject' object has no attribute 'switch_id'". What could be the reason? 

Log: 
2015/02/15 18:40:16 nbapi: nbapi_module_init 
2015/02/15 18:40:16 nbapi: [mul-core] service instance created 
2015/02/15 18:40:16 nbapi: mul_route_service_get: 
shm_open: Success 
2015/02/15 18:40:16 nbapi: No such service [mul-fab-cli] 
2015/02/15 18:40:16 nbapi: nbapi_module_init: Mul fab service instantiation failed 
2015/02/15 18:40:16 nbapi: [mul-tr] service instance created 
2015/02/15 18:40:16 nbapi: No such service [mul-makdi] 
2015/02/15 18:40:16 nbapi: nbapi_module_init: Mul makdi service instantiation failed 
2015/02/15 18:40:16 nbapi: [infra] switch 0x1 add 
2015/02/15 18:40:18 nbapi: INFO [tornado] preparing http server 
2015/02/15 18:40:18 nbapi: INFO [tornado] Starting API server on port 8181 
2015/02/15 18:40:36 BaseHandler DEBUG pattern: ^0x[0-9a-fA-F]+/limit$, url: 
2015/02/15 18:40:36 BaseHandler DEBUG pattern: ^$, url: 
2015/02/15 18:40:36 SwitchHandler DEBUG matched func: get_switch_all, args: (None, None) 
2015/02/15 18:40:36 tornado.application ERROR Uncaught exception GET /1.0/topology/switch (127.0.0.1) 
HTTPRequest(protocol='http', host='127.0.0.1:8181', method='GET', uri='/1.0/topology/switch', version='HTTP/1.1', remote_ip='127.0.0.1', headers={'Host': '127.0.0.1:8181', 'Accept': '*/*', 'User-Agent': 'curl/7.37.1'}) 
Traceback (most recent call last): 
File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1218, in _when_complete 
callback() 
File "/usr/local/lib/python2.7/dist-packages/tornado/web.py", line 1239, in _execute_method 
self._when_complete(method(*self.path_args, **self.path_kwargs), 
File "/home/user/Documents/Openflow/Controllers/Mul/Source/openmul-master-old/application/nbapi/py-tornado/app/handler/switch.py", line 39, in get 
self.__execute(dpid, port_no) 
File "/home/user/Documents/Openflow/Controllers/Mul/Source/openmul-master-old/application/nbapi/py-tornado/app/handler/switch.py", line 51, in __execute 
getattr(self, func)(*args) 
File "/home/user/Documents/Openflow/Controllers/Mul/Source/openmul-master-old/application/nbapi/py-tornado/app/handler/switch.py", line 59, in get_switch_all 
res = self.__nbapi_switch_brief_list_t_serialization(res) 
File "/home/user/Documents/Openflow/Controllers/Mul/Source/openmul-master-old/application/nbapi/py-tornado/app/handler/switch.py", line 129, in __nbapi_switch_brief_list_t_serialization 
dpid = int(s.switch_id.datapath_id) 
AttributeError: 'SwigPyObject' object has no attribute 'switch_id' 
2015/02/15 18:40:36 tornado.access ERROR 500 GET /1.0/topology/switch (127.0.0.1) 205.86ms 
swig/python detected a memory leak of type 'c_ofp_switch_brief_t *', no destructor found.

**Comment by openmul** [comment]
I did a quick test to verify: 

1. Cloned the latest openmul master branch 
2. Ran all the necessary modules - mul (core), multr, mulnbapi 
3. In local browser, opened the following: 

http://127.0.0.1:8181/1.0/topology 
http://127.0.0.1:8181/1.0/topology/switch 

It showed values properly. 

4. Ran test program in openmul/application/webapp/openmul_switch_get_webapp.py 

All steps worked fine and I could not find any issues. My test system is Ubuntu 12.04.3 LTS x86-64. Need to triage on 14.10 x86 !!
